build/NuSpecs/UmbracoCms.Core.nuspec

```diff
@@ -28,17 +28,17 @@
       <dependency id="MySql.Data" version="[6.9.8, 7.0.0)" />
       <dependency id="xmlrpcnet" version="[2.5.0, 3.0.0)" />
       <dependency id="ClientDependency" version="[1.9.2, 2.0.0)" />
-      <dependency id="ClientDependency-Mvc5" version="[1.8.0, 2.0.0)" />
+      <dependency id="ClientDependency-Mvc5" version="[1.8.0.0, 2.0.0)" />

```

Comment:
I just re-installed it..same thing. It's the NuPkg that has that version: 

![image](https://cloud.githubusercontent.com/assets/304656/22727218/0ef9d6c2-edd8-11e6-9cee-94bf6439bd3b.png)